experiments/Pulse_Switching_nTron.py

```diff
@@ -150,7 +150,7 @@ def setup_arb(self): #, gate_bias, gate_pulse_amplitude, gate_pulse_duration):
         seg_ids_ch1.append(seg_id)
 
         # Build in a delay between sequences
-        settle_pts = int(640*np.ceil(self.repeat_time * self.sample_rate / 640))
+        settle_pts = int(640*np.int(np.ceil(self.repeat_time * self.sample_rate / 640)))
```

Comment:
True. I was distracted by the fact that np.ceil returns floats, against my expectations...